### ISSUE TITLE
Incorrect Version for Code fonts in recent release

### ISSUE THREAD

**Issue by bevanjkay** [arrowtype/recursive#474]
**Problem description**
In the latest version `1.079` the font files inside the Recursive_Code folder are still versioned with `1.078`.
They may have not been updated in the latest release, however this is causing the version bump to fail for Homebrew as the font file cannot be located during install.

https://github.com/Homebrew/homebrew-cask-fonts/pull/4793

**Font paths;**
```
"ArrowType-Recursive-1.079/Recursive_Code/RecMonoCasual/RecMonoCasual-Bold-1.078.ttf"
"ArrowType-Recursive-1.079/Recursive_Code/RecMonoCasual/RecMonoCasual-BoldItalic-1.078.ttf"
"ArrowType-Recursive-1.079/Recursive_Code/RecMonoCasual/RecMonoCasual-Italic-1.078.ttf"
"ArrowType-Recursive-1.079/Recursive_Code/RecMonoCasual/RecMonoCasual-Regular-1.078.ttf"
"ArrowType-Recursive-1.079/Recursive_Code/RecMonoDuotone/RecMonoDuotone-Bold-1.078.ttf"
"ArrowType-Recursive-1.079/Recursive_Code/RecMonoDuotone/RecMonoDuotone-BoldItalic-1.078.ttf"
"ArrowType-Recursive-1.079/Recursive_Code/RecMonoDuotone/RecMonoDuotone-Italic-1.078.ttf"
"ArrowType-Recursive-1.079/Recursive_Code/RecMonoDuotone/RecMonoDuotone-Regular-1.078.ttf"
"ArrowType-Recursive-1.079/Recursive_Code/RecMonoLinear/RecMonoLinear-Bold-1.078.ttf"
"ArrowType-Recursive-1.079/Recursive_Code/RecMonoLinear/RecMonoLinear-BoldItalic-1.078.ttf"
"ArrowType-Recursive-1.079/Recursive_Code/RecMonoLinear/RecMonoLinear-Italic-1.078.ttf"
"ArrowType-Recursive-1.079/Recursive_Code/RecMonoLinear/RecMonoLinear-Regular-1.078.ttf"
"ArrowType-Recursive-1.079/Recursive_Code/RecMonoSemicasual/RecMonoSemicasual-Bold-1.078.ttf"
"ArrowType-Recursive-1.079/Recursive_Code/RecMonoSemicasual/RecMonoSemicasual-BoldItalic-1.078.ttf"
"ArrowType-Recursive-1.079/Recursive_Code/RecMonoSemicasual/RecMonoSemicasual-Italic-1.078.ttf"
"ArrowType-Recursive-1.079/Recursive_Code/RecMonoSemicasual/RecMonoSemicasual-Regular-1.078.ttf"
```

**Comment by arrowtype** [comment]
Ohhh shoot, thanks for catching that! I'll have to look into this ASAP. I think they are actually the 1.079 files, but probably just mislabeled. Will update soon!

**Comment by arrowtype** [comment]
Ahh, okay, checking this more closely, I now understand the issue (not that the original post wasn’t clear – I was just on my phone and distracted). It was only the Code fonts that were the incorrect version. I have now corrected that and updated the release as a hotfix (i.e. I replaced the files with correct ones, but didn’t make a whole new point version release).

Thanks again for spotting that and taking the time to let me know about it, @bevanjkay! Hope you’re enjoying the fonts. :-)
